src/lava/utils/float2fixed.py

```diff
@@ -1,26 +1,719 @@
 # Copyright (C) 2021-22 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 # See: https://spdx.org/licenses/
-
 """
-This module will contain a tool to perform automated float- to fixed-point
-conversion of of Lava Processes.
-
-Given a set of processes and data, the tool will perform the following steps:
-
-1. Find fixed-point ProcessModels of every floating-point ProcessModel if
-available and throw an error otherwise.
-2. Run the floating-point model and collect activation statistics.
-3. Given activation statistic and type/precision/dynamic range specifications
-of fixed-point models, perform input parameter conversion for fixed-point model.
-4. Replace floating-point ProcessModels by corresponding fixed-point
-ProcessModels
-5. Initialize fixed-point ProcessModels
-
-ProcessModel extraction might be a shared function with compiler.
-
-This will likely lead to some performance degradation of the application so
-perhaps some iterative optimization may have to be included within this
-parameter conversion process. Reuse other libraries for this purpose where
-possible.
- """
+This module containx a tool to perform automated float- to fixed-point
+conversion of Lava Processes.
+"""
+from lava.magma.core.run_configs import RunConfig
+from lava.magma.core.run_conditions import RunSteps
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.compiler.compiler_graphs import (
+    find_processes,
+    ProcGroupDiGraphs)
+from lava.magma.core.process.variable import Var
+from lava.magma.core.process.ports.ports import AbstractSrcPort
+from lava.proc.monitor.process import Monitor
+from lava.magma.core.process.ports.ports import OutPort
+
+import numpy as np
+import inspect
+
+
+class Float2FixedConverter:
+
+    def __init__(self):
+        self.floating_pt_rcfg = None
+        self.fixed_pt_rcfg = None
+        self.procs = None
+        self.hierarchical_procs = None
+        self.run_proc = None
+        self.var_ports = None
+        self.fixed_pt_proc_models = None
+        self.num_steps = None
+        self.quantiles = None
+        self.scale_domains = set()
+
+    def set_run_cfg(self, floating_pt_rcfg, fixed_pt_rcfg) -> None:
```

Comment:
I didn't see where set_run_cfg is being used.
But maybe it's easier/cleaner to simply write setter methods for floating_pt_rcfg und fixed_pt_rcfg?
https://www.geeksforgeeks.org/getter-and-setter-in-python/

Reply:
`set_run_cfg` is used be the user before calling the `converter.convert`, see [here, cell 11](https://github.com/ackurth-nc/lava/blob/dev/f2f_conv/tutorials/end_to_end/tutorial02_oxford_input_transformation.ipynb).